client/views/admin/emailInbox/EmailInboxForm.js

```diff
@@ -0,0 +1,361 @@
+import React, { useCallback, useState } from 'react';
+import { useMutableCallback } from '@rocket.chat/fuselage-hooks';
+import {
+	Accordion,
+	Button,
+	ButtonGroup,
+	TextInput,
+	TextAreaInput,
+	Field,
+	ToggleSwitch,
+	FieldGroup,
+	Box,
+	Margins,
+} from '@rocket.chat/fuselage';
+
+import { AutoCompleteDepartment } from '../../../components/AutoCompleteDepartment';
+import { useTranslation } from '../../../contexts/TranslationContext';
+import { useRoute } from '../../../contexts/RouterContext';
+import { useToastMessageDispatch } from '../../../contexts/ToastMessagesContext';
+import Page from '../../../components/Page';
+import { useForm } from '../../../hooks/useForm';
+import { useEndpointAction } from '../../../hooks/useEndpointAction';
+import { isEmail } from '../../../../app/utils';
+import { useEndpointData } from '../../../hooks/useEndpointData';
+import { AsyncStatePhase } from '../../../hooks/useAsyncState';
+import { FormSkeleton } from './Skeleton';
+import DeleteWarningModal from '../../../components/DeleteWarningModal';
+import { useSetModal } from '../../../contexts/ModalContext';
+import { useComponentDidUpdate } from '../../../hooks/useComponentDidUpdate';
+
+
+const initialValues = {
+	active: true,
+	name: '',
+	email: '',
+	description: '',
+	senderInfo: '',
+	department: '',
+	// SMTP
+	smtpServer: '',
+	smtpPort: '',
+	smtpUsername: '',
+	smtpPassword: '',
+	smtpSecure: false,
+	// IMAP
+	imapServer: '',
+	imapPort: '',
+	imapUsername: '',
+	imapPassword: '',
+	imapSecure: false,
+};
+
+const getInitialValues = (data) => {
+	if (!data) {
+		return initialValues;
+	}
+
+	const {
+		active,
+		name,
+		email,
+		description,
+		senderInfo,
+		department,
+		smtp,
+		imap,
+	} = data;
+
+	return {
+		active: active ?? true,
+		name: name ?? '',
+		email: email ?? '',
+		description: description ?? '',
+		senderInfo: senderInfo ?? '',
+		department: department ?? '',
+		// SMTP
+		smtpServer: smtp.server ?? '',
+		smtpPort: smtp.port ?? '',
```

Comment:
I think we could use the default used port here as int.
```suggestion
		smtpPort: smtp.port ?? 587,
```